Formula/llvm.rb

```diff
@@ -177,6 +178,12 @@ def install
     # install llvm python bindings
     (lib/"python2.7/site-packages").install buildpath/"bindings/python/llvm"
     (lib/"python2.7/site-packages").install buildpath/"tools/clang/bindings/python/clang"
+
+    # install emacs modes
+    elisp.install Dir["utils/emacs/*.el"]
+    elisp.install buildpath/"tools/clang/tools/clang-format/clang-format.el"
+    elisp.install buildpath/"tools/clang/tools/clang-rename/clang-rename.el"
+    elisp.install buildpath/"tools/clang/tools/extra/clang-include-fixer/tool/clang-include-fixer.el"
```

Comment:
```suggestion
    elisp.install Dir["utils/emacs/*.el"] + %w[
      tools/clang/tools/clang-format/clang-format.el
      tools/clang/tools/clang-rename/clang-rename.el
      tools/clang/tools/extra/clang-include-fixer/tool/clang-include-fixer.el
    ]
```

Reply:
(or, if this doesn't quite work, something similar) that avoids the duplicated `buildpath` and `elisp.install`